### PR TITLE
Unify config

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- [x] Change Config to be a subclass of Config from packit.
- [x] Use the new authentication method.
- [x] Get the project objects via `config.get_project(url)`.
~- [ ] backward-compatibility (new authentication)~

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
recheck

File: packit_service/service/events.py [hunk 1]
```diff
@@ -418,6 +396,4 @@ def get_package_config(self):
         return package_config
 
     def get_project(self) -> GitProject:
```
Comment (rpitonak) [inline packit_service/service/events.py]: could we put this method to `AbstractGithubEvent` as it is the same for all of the Github Events?

**Comment by lachmanfrantisek** [comment]
> PS: I still see one item unchecked in PR's TODO list.

That's correct if we do not want to change the structure of the production config.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -152,3 +152,14 @@ Once we have jobs defined in `packit` config we are ready to move on to next ste
...
 Thank you for your interest!
 packit team.
+
+### Service configuration
+
+The service configuration is an extension of the user configuration from packit. (`Config` class in [packit/config.py](https://github.com/packit-service/packit/blob/master/packit/config.py).)
+
+To add a new service-related property you need to:
+
+1. Add a property to `ServiceConfig.__init__` in [packit/config.py](https://github.com/packit-service/packit/blob/master/packit/config.py).
```
Comment (jpopelka) [inline CONTRIBUTING.md]: Sorry for being late for the party, but #154

**Comment by TomasTomecek** [comment]
` +210 −327` very nice!

**Pull request merged**